wormhole-connect/src/config/mainnet.ts

```diff
@@ -116,6 +127,7 @@ export const MAINNET_TOKENS: { [key: string]: TokenConfig } = {
     decimals: 18,
     solDecimals: 8,
     suiDecimals: 8,
+    aptosDecimals: 8,
```

Comment:
Maybe it's time to refactor this. We could have a single `decimals` property that's the token's decimals on its native chain. Then we could have a function that returns the token's decimals for a given chain.

Reply:
How do Aptos decimals work? solana and sui are 9 for native and max 8 for everything else right?

Reply:
Yeah, for Aptos the decimals are 8 for native and capped at 8 for everything else